antarest/study/storage/utils.py

```diff
@@ -81,10 +82,11 @@ def update_antares_info(metadata: Study, studytree: FileStudyTree) -> None:
 
 def fix_study_root(study_path: Path) -> None:
     """
-    Fix possibly the wrong study root on zipped archive (when the study root is nested)
+    Fix possibly the wrong study root in zipped archive (when the study root is nested)
 
     @param study_path the study initial root path
     """
+    # TODO: what if it is a zipped output ?
```

Comment:
remove todo, it won't be called on a zipped output

Reply:
Things changed a bit. It will be called, and will do nothing